website/docs/r/dns_custom_resolver_location.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "DNS Services"
+layout: "ibm"
+page_title: "IBM : ibm_dns_cr_locations"
+description: |-
+  Manages IBM Private DNS Custom Resolver Locations.
+---
+
+# ibm_dns_cr_locations
+
+Provides a private DNS Custom Resolver Locations resource. This allows DNS Custom Resolver Location to create, update, and delete. 
+
+
+## Example usage
+
+```terraform
+resource "ibm_dns_cr_locations" "test" {
```

Comment:
Earlier cr was spelled out custom_resolver.  This resource is going to create 1 location.  A better name would be ibm_dns_custom_resolver_location

the ibm_dns_custom_resolver resource creates a ibm_dns_custom_resolver_location (in fact it is required).   I've found that creating resources in the parent, and creating resources directly  causes problems.  @hkantare - thoughts?